src/utils/search-query-transform.ts

```diff
@@ -1,11 +1,21 @@
+// @ts-ignore
```

Comment:
Idk why this file doesn't have a Typescriptification issue but I feel like you should get credit for converting it still! :tada: